src/accelerate/commands/config/cluster.py

```diff
@@ -486,6 +502,8 @@ def get_cluster_input():
         machine_type = str(distributed_type).split(".")[1].replace("MULTI_", "")
         if machine_type == "TPU":
             machine_type += " cores"
+        elif machine_type == "CPU":
+            machine_type = "processes"
```

Comment:
```suggestion
            machine_type = " processes"
```

Reply:
If I make this change, there ends up being an extra space in the prompt like:
`How many  processes should be used for distributed training?`